### PR TITLE
Added Atom's One theme

### PR DESCRIPTION
**Pull request by Th3Whit3Wolf**
This theme has a light and dark mode.

### CODE REVIEW

**Review by glepnir — COMMENTED** [review]
Can you provide a image ?

File: autoload/spaceline/colorscheme/one.vim [hunk 1]
```diff
@@ -0,0 +1,121 @@
+" =============================================================================
+" Filename: spaceline.vim
+" Author: taigacute
```
Comment (glepnir) [inline autoload/spaceline/colorscheme/one.vim]: change the author name

**Review by glepnir — COMMENTED** [review]
change the information

**Comment by Th3Whit3Wolf** [comment]
spaceline_one_light

**Comment by glepnir** [comment]
Merged Thanks.

**Pull request merged**